Makefile

```diff
@@ -25,6 +25,12 @@ bin:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' github.com/vmware-tanzu/antrea/cmd/...
 
+.PHONY: windows-bin
+windows-bin:
+	@mkdir -p $(BINDIR)
+	GOOS=windows $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' github.com/vmware-tanzu/antrea/cmd/antrea-cni \
```

Comment:
for antctl, I believe we had an ".exe" suffix. Should we do the same for these?

Reply:
Yes we should have the ".exe" suffix.

Fortunately, the `go build` will add the suffix automatically:  `The ".exe" suffix is added when writing a Windows executable`. So think we don't need to add the suffix manually.

Reply:
I see the difference, antctl use `build -o $(BINDIR)/$@` which specifies the bin name instead of `build -o $(BINDIR)` which lets go build decide the bin name.